sdcm/nemesis/__init__.py

```
@@ -5297,14 +5308,12 @@ def disrupt_kill_mv_building_coordinator(self):
                 with adaptive_timeout(operation=Operations.CREATE_MV, node=working_node, timeout=14400) as timeout:
                     wait_for_view_to_be_built(working_node, ks_name, view_name, timeout=timeout * 2)
 
-                with self.cluster.cql_connection_patient(node=working_node, connect_timeout=600) as session:
-                    result = list(
-                        session.execute(SimpleStatement(f"SELECT * FROM {ks_name}.{view_name} limit 1", fetch_size=10))
-                    )
-                    assert len(result) >= 1, f"MV {ks_name}.{view_name} was not built"
+                result = list(
+                    session.execute(SimpleStatement(f"SELECT * FROM {ks_name}.{view_name} limit 1", fetch_size=10))
```

Ruff (0.15.18)

[error] Possible SQL injection vector through string-based query construction

(S608)